packages/frontapp/src/app/components/UserList/index.tsx

```diff
@@ -110,13 +117,16 @@ const UserList: FC<IProps> = ({ itemsTaken, sortBy }) => {
   const filterUsers = (category: string) => {
```

Comment:
Not sure if my code works, but it seems that here we can remove the `switch/case` statements:
```JavaScript
const categorySortsMap: Record<string, SortType> = {
    'By alphabet': sortAlphabetically,
    'Number of things taken': sortByThingsTaken,
    'Number of overdue deals': sortByThingsOverdue,
};
const filterUsers = (category: keyof typeof categorySortsMap) => {
    if (searchReaders.length) {
        setReaders([...searchReaders].sort(categorySortsMap[category]));
    }
};
```

Reply:
Didn't want to touch the previous code to not break anything, but I'll give it a try, thanks